database/seeders/RolePermissionSeeder.php

```diff
@@ -73,11 +73,13 @@ public function run()
         RolePermission::insert(['role_id' => '6', 'permission_id' => 45]);
 
         /***** Sector role permissions *****/
+        RolePermission::insert(['role_id' => '7', 'permission_id' => 41]);
         RolePermission::insert(['role_id' => '7', 'permission_id' => 46]);
         RolePermission::insert(['role_id' => '7', 'permission_id' => 47]);
 
         /***** IPQMSO role permissions *****/
-        RolePermission::insert(['role_id' => '8', 'permission_id' => 48]);
+        RolePermission::insert(['role_id' => '8', 'permission_id' => 41]);
+        RolePermission::insert(['role_id' => '8php ', 'permission_id' => 48]);
```

Comment:
Please modify this part

Reply:
php in line 82 has been removed. Thank you.